meet_balls_app/views.py

```diff
@@ -127,8 +127,10 @@ def edit_profile(request):
     if request.method == 'POST':
         birth_date = request.POST.get('birth_date')
         favorite_ball_game = request.POST.get('favorite_ball_game')
+        profile_picture = request.FILES.get('profile_picture')
+        print(profile_picture)
```

Comment:
please remove this print

Reply:
removed, it's leftovers from serious debug process last night